spec/tasks/scripts/delete_bangladesh_demo_facility_spec.rb

```diff
@@ -0,0 +1,66 @@
+require "rails_helper"
+require "tasks/scripts/delete_bangladesh_demo_facility"
+
+RSpec.describe DeleteBangladeshDemoFacility do
+  describe ".handle_patients" do
+    it "moves the designated patients to Beanibazar and discards the rest" do
+      moveable_patient_ids = %w[
```

Comment:
Could just do `DeleteBangladeshDemoFacility::REAL_PATIENT_IDS` here?

Reply:
I didn't want the test to be self-fulfilling 😅  If the list of IDs changes for any reason, I want to make sure that we're aware of it through a failing test.

Reply:
Ah, gotcha, this is a regression list. Makes sense 👍